src/doc/style/README.md

```diff
@@ -0,0 +1,77 @@
+% Rust Guidelines [working title]
+
+This document collects the emerging principles, conventions, abstractions, and
+best practices for writing Rust code.
+
+Since Rust is evolving at a rapid pace, these guidelines are
+preliminary. The hope is that writing them down explicitly will help
+drive discussion, consensus and adoption.
+
+Whenever feasible, guidelines provide specific examples from Rust's standard
+libraries.
+
+For now, you can find a rendered snapshot at
+[http://aturon.github.io/](http://aturon.github.io/).  After
+[some infrastructure work](https://github.com/aturon/rust-guidelines/issues/17), the snapshot will move somewhere more
+official.
+
+### Building the document
+
+Like http://rustbyexample.com/, this guidelines document is written in
+the [`gitbook`](https://github.com/GitbookIO/gitbook) style. It can be
+compiled with a prototype tool,
+[`rustbook`](https://github.com/aturon/rust-book) that provides a
+minimal subset of `gitbook`'s functionality on top of `rustdoc`.
```

Comment:
Hm, this probably needs to be edited and it may need a general pass as well in more places, but that may be able to wait.